pkg/vm/engine/disttae/txn_table.go

```diff
@@ -681,6 +687,8 @@ func (tbl *txnTable) Ranges(ctx context.Context, exprs []*plan.Expr) (ranges eng
 		exprs,
 		&blocks,
 		tbl.proc.Load(),
+		txnOffset,
+		fromSnapshot,
```

Comment:
Pls remove `fromSnapshot`

Reply:
> Pls remove `fromSnapshot`



> Pls remove `fromSnapshot`



> Pls remove `fromSnapshot`

has deleted the newly added frmSnapshot parameter in this PR @triump2020